sccm/develop/core/understand/windows-powershell-basics.md

```diff
@@ -17,9 +17,9 @@ System Center 2012 Configuration Manager SP1 introduced native PowerShell suppor
 
  To get started:  
 
-1.  Hit your Windows key and type “PowerShell��?.  
+1.  Hit your Windows key and type "PowerShell".  

```

Comment:
Thanks! think there are a bunch of smartquote/encoding issues in the repo if you do a search